CHANGES.rst

```diff
@@ -179,6 +179,10 @@ Bug fixes
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- ``astropy.io.ascii``
+
+  - Update some function docstrings to the Numpy format [#3352]
+
```

Comment:
This can go in the 1.0.2 release.  It doesn't really need a changelog entry at all, but you can put it in if you want--you'll just have to move it to the right section.  Up to you.